.gitignore

```diff
@@ -6,3 +6,4 @@ flamegraph.svg
 /.markdownlint.json
 /.rustfmt.toml
 /.yamllint.yaml
+.DS_Store
```

Comment:
@marschattha you can configure your Git to ignore .DS_Store files everywhere on your system, I think

Reply:
Oh yea this wasn't supposed to be commited